### PR TITLE
Adds some http example files to worksheets to make testing easier

### PR DESCRIPTION
**Pull request by dbbaughe**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: worksheets/rollups/comparisons.http [hunk 1]
```diff
@@ -0,0 +1,198 @@
+###
+
+#21, 14, 3, 11, 24, 1, 1, 41, 14, 13
```
Comment (qreshi) [inline worksheets/rollups/comparisons.http:3]: What is this used for?

File: worksheets/rollups/create.http [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  "rollup": {
+    "enabled": true,
+    "schedule": {
+      "interval": {
+        "period": 1,
+        "unit": "Minutes",
+        "start_time": {{$timestamp}}
+      }
+    },
+    "last_updated_time": {{$timestamp}},
```
Comment (qreshi) [inline worksheets/rollups/create.http:15]: Is anything strange going on here with the formatting for there to be syntax highlighting on some of the `{$timestamp}` and `,`?

File: worksheets/rollups/comparisons.http (hunk 1 above)
Comment (dbbaughe) [reply worksheets/rollups/comparisons.http:3]: A secret code

I kid I kid, probably some numbers I was keeping track of for comparing raw rollup data vs rollup data in below queries.

File: worksheets/rollups/create.http (hunk 2 above)
Comment (dbbaughe) [reply worksheets/rollups/create.http:15]: Github just doesn't read the {{$timestamp}} syntax correctly

File: worksheets/rollups/comparisons.http (hunk 1 above)
Comment (thalurur) [reply worksheets/rollups/comparisons.http:3]: are you removing these Drew?

Comment (dbbaughe) [reply worksheets/rollups/comparisons.http:3]: I wasn't planning on it as it's used for comparison in the query below, but I can if someone feels strongly about it.
Again though I wouldn't look *too* much into this PR. These are just sample queries added to be used when running a local cluster for some quick sanity checks among other things.

**Review by qreshi — APPROVED** [review]

**Review by bowenlan-amzn — APPROVED** [review]

**Pull request merged**